index.html

```diff
@@ -53,22 +53,20 @@
           Here is a simple form:
         </p>
         <form method="get" action="#">
-          <fieldset>
-            <label for="email">Your email:</label>
-            <input name="email" type="text" id="email" size="25" placeholder="must be unique"/>
-            <label for="sex">Your gender:</label>
-            <select name="sex" id="sex">
-              <option value="none">won't tell</option>
-              <option value="female">female</option>
-              <option value="male">male</option>
-            </select>
-            <label for="address">Your full address:</label>
-            <textarea name="address" id="address" cols="40" rows="3" placeholder="PO box is not allowed"></textarea>
-            <input type="checkbox" name="consent" id="consent" value="test"/>
-            <label for="consent">I agree with all possible terms and conditions</label>
-            <button type="submit">Subscribe</button>
-            <button type="reset">Reset</button>
-          </fieldset>
+          <label for="email">Your email:</label>
```

Comment:
hm... I don't think it's a good idea to remove `fieldset`. I would actually add `<legend>` to it, instead of removing: http://www.w3schools.com/tags/tag_legend.asp